src/index.html

```diff
@@ -26,7 +30,7 @@ <h3>Add Your score</h3>
       </form>
     </div>
   </div>
-  <footer><p>LeaderBoard ©2022 (Petro Lesapiti)</p></footer>
-  <script type="module" src="index.js"></script>
+  <footer><p> © WORLD ATLETICS 2022</p></footer>
+  <script src="index.js"></script>
```

Comment:
- [ ] Kindly remove the link to the JS file as the webpack already knows the entry point file.